tiered-storage/jcloud/src/test/java/org/apache/bookkeeper/mledger/offload/jcloud/BlobStoreTestBase.java

```diff
@@ -69,8 +88,9 @@ public void start() throws Exception {
     @AfterMethod
     public void tearDown() {
         if (blobStore != null &&
-            (!Boolean.parseBoolean(System.getProperty("testRealGCS", "false")) &&
-             !Boolean.parseBoolean(System.getProperty("testRealGCS", "false")))) {
+            (Boolean.parseBoolean(System.getProperty("testRealAWS", "false")) ||
```

Comment:
This change seems not right, for real test, we should not call delete container.